docs/layers.md

```diff
@@ -1,116 +1,510 @@
 
-Properties:
+Need help displaying vector tiles in a map? We have several [examples](display-tiles.md) using Mapzen vector tiles to style in your favorite graphics library including Tangram, Mapbox GL, D3, and Hoverboard.
+
+## Overview
+
+#### Data sources and attribution
+
+Mapzen primarily sources from OpenStreetMap, but includes a variety of other open data. For a full listing, view the [data sources](data-sources.md). Each source may require [attribution](attribution.md) in your project.
+
+#### Name localization
+
+Mapzen vector tile features include all source names in the default `name` property, and variants in the `name:*` properties. When different than `name`, we also include `alt_name:*`, `alt_name_`, `old_name:*` values to enable full internationalization. Tangram supports all scripts.
+
+For features in the Boundaries layer, we support two additional variants `left:name:*` and `right:name:*` to support oriented labeling on the appropriate side of the boundary line (so the labeled polygon's text can appear inside that polygon consistently).
+
+#### Geometry types
+
+Individual Mapzen vector tile layers can include mixed geometry types. This is common in the landuse, water, and building layers.
+
+A tile geometry can be one of 3 types:
+
+* Point, MultiPoint
+* LineString, MultiLineString
+* Polygon, MultiPolygon
+
+In Tangram, Mapzen's GL graphics library, the keyword `$geometry` matches the feature's geometry type, for cases when a FeatureCollection includes more than one type of kind of geometry. Valid geometry types are:
+
+* `point`: matches Point, MultiPoint
+* `line`: matches LineString, MultiLineString
+* `polygon`: matches Polygon, MultiPolygon
+
+**Tangram scene file examples:**
+
+```
+filter: { $geometry: polygon }                      # matches polygons only
+filter: { $geometry: [point, line] }                # matches points & lines
+filter: function() { return $geometry === 'line' }  # matches lines only
+```
+
+Mapnik supports geometry filtering via the special `mapnik::geometry_type` keyword.
+
+**CartoCSS Examples:**
+
+```
+#layer['mapnik::geometry_type'=1] { /* point styles */ }
+#layer['mapnik::geometry_type'=2] { /* line styles */ }
+#layer['mapnik::geometry_type'=3] { /* polygon styles */ }
+```
+
+
+#### Data updates
+
+Most Mapzen vector tile content is updated minutely from OpenStreetMap. Low and mid-zoom tiles are updated approximately monthly. Some source data rarely updates – Natural Earth updates approximately yearly.
+
+#### Changelog
+
+The current version of Mapzen vector tiles is v0.7. Our tiles are still in active development, but we try to minimize backwards incompatable breaking changes. We're also interesting in your feedback at hello@mapzen.com!
```

Comment:
Should we embed the version in the docs like this? Does it add anything to do the docs, given that the version is not part of the URL, and has no explicit user impact. Also, I think we might forget to update it. Perhaps we could embed the tag, or an <img> element which shows the currently-deployed tag?